tool/deploy.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:io';
+
+import 'package:grinder/grinder.dart';
+import 'package:pub_semver/pub_semver.dart';
+
+main(List args) => grind(args);
+
+@DefaultTask('Deploy software')
+deploy() {
```

Comment:
Is it possible to expose new tasks to grinder from other Dart files? After a certain point navigating `tool/grind.dart` will become difficult it its a large file.